packages/cli/src/utils.ts

```diff
@@ -391,3 +393,35 @@ export async function loadConfigAndHandleErrors(
     return new Config({ apis: {}, styleguide: {} });
   }
 }
+
+export function sorTopLevelKeysForOas(
+  document: Oas3Definition | Oas2Definition
+): Oas3Definition | Oas2Definition {
+  if ('swagger' in document) {
+    // do we need sort keys for oas2 ?
+    return document;
```

Comment:
yes, let's sort them too. I can help to define the order.

Reply:
Okay, please share your variant of the order

Reply:
Let's use like here: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#swagger-object

But move `security`, `tags` and `externalDocs` after `produces`.